<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B4E308O" failures="1" tests="1" name="AccountPageTest" time="39.83" errors="0" timestamp="2021-09-02T11:08:10 IST">
  <testcase classname="com.qaautomation.tests.AccountPageTest" name="DoVerifyUsername" time="25.479">
    <failure type="java.lang.AssertionError" message="expected [ZhKFOSyXDO] but found [zhKFOSyXDO]">
      <![CDATA[java.lang.AssertionError: expected [ZhKFOSyXDO] but found [zhKFOSyXDO]
at com.qaautomation.tests.AccountPageTest.DoVerifyUsername(AccountPageTest.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- DoVerifyUsername -->
</testsuite> <!-- AccountPageTest -->
